packages/ketcher-core/src/application/render/renderers/BaseMonomerRenderer.ts

```diff
@@ -91,6 +91,14 @@ export abstract class BaseMonomerRenderer extends BaseRenderer {
     );
   }
 
+  protected getMonomerColor(theme) {
+    return (
+      theme.monomer.color[
+        this.monomer.monomerItem.props.MonomerNaturalAnalogCode
+      ]?.regular || theme.monomer.color.undefined
```

Comment:
Can we change a name from `theme.monomer.color.undefined` to something else? I bet it can confuse people and look like an error. 
May be something like `notKnown`, `default` or something?